source/manual/github.html.md

```diff
@@ -7,21 +7,76 @@ section: GitHub
-[`govuk`](https://github.com/search?q=topic:govuk).
+# GitHub organisation
 
-If you are unable to access an alphagov repository you can contact the
-[GDS Github Owners Google group](https://groups.google.com/a/digital.cabinet-office.gov.uk/g/gds-github-owners), where members of that group have superadmin access to give you permissions to
-contribute to a repo.
+Our GitHub organisation is called [alphagov][https://github.com/alphagov]. We (GOV.UK) share it with other teams in the Government Digital Service (GDS).
 
-We use [govuk-saas-config][] to configure our repositories.
+The organisation is on the Enterprise Cloud plan, which grants us access to GitHub's enterprise support. Only [people with enterprise permissions](https://github.com/orgs/alphagov/people/enterprise_owners) can access enterprise support ([limited to enterprise owners and up to 20 additional members](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-accounts-and-repositories/managing-users-in-your-enterprise/managing-support-entitlements-for-your-enterprise)).
 
-Also see the [GOV.UK reliance on GitHub features doc](https://docs.google.com/document/d/1KsYWCHSQZEwqB2NF1A7Z9rmP1s1azcZhEamaPsXAoxk/edit#heading=h.75pp7jqw5sk9) for more information.
+[GDS GitHub Owners](https://groups.google.com/a/digital.cabinet-office.gov.uk/g/gds-github-owners) have superadmin access to alphagov. You can contact them to request changes to [organisation settings](https://docs.github.com/en/organizations/managing-organization-settings), or to request access to an inaccessible repository (e.g. one that was created by an ex-employee).
 
-[alphagov]: https://github.com/alphagov
-[govuk-saas-config]: https://github.com/alphagov/govuk-saas-config/tree/master/github
+# GOV.UK teams
+
+There are several GOV.UK GitHub teams within alphagov, including:
+
+- [GOV.UK][team-govuk].
+  Grants write access to GOV.UK repos, as well as integration admin access to the CI environment.
+
+- [GOV.UK Production Deploy][team-govuk-production-deploy].
+  Grants the ability to merge PRs against continuously deployed apps, and the ability to [deploy apps to staging and production](/manual/rules-for-getting-production-access.html#production-deploy-access)
+
+- [GOV.UK Production Admin][team-govuk-production-admin].
+  Grants admin access to GOV.UK repos, and [general admin access to staging and production](/manual/rules-for-getting-production-access.html#production-admin-access).
+
+- [GOV.UK Content Designers][team-covuk-content-designers].
+  This team allows the user to create a branch in a GOV.UK repository and open a pull request, but only a developer can merge the request.
+  Only content designers should be added to this team.
+
+# Getting access to GitHub
+
+Not everyone on GOV.UK requires GitHub access, as much of what we do is in the open. However, if your role requires it, you can be added to the org and the relevant team(s) through Terraform, in [govuk-user-reviewer][].
```

Comment:
Nit:

Might be worth adding a note to reinforce that we must not do this manually. (As it breaks adding permission later in Terraform)

Reply:
Added a warning 👍